apps/frontend/app/[locale]/(unauthenticated)/page.tsx

```diff
@@ -140,6 +141,13 @@ export default async function Home() {
       </CanUserServer>
       <h3 className="mt-4 mb-2 font-head text-3xl">Components</h3>
       <div className="flex flex-col gap-2 px-4 pb-4">
+        <WysiwygEditor
+          placeholder="Write your description here..."
+          onChange={(editorState) =>
+            console.log('Changed WYSIWYG', editorState)
+          }
```

Comment:
Die Funktion macht bei mir Probleme (siehe Fehlermeldung). Funktioniert aber meiner Ansicht nach auch ohne (?) - ich hab es auskommentiert und der Editor funktioniert gut.

Fehlermeldung: [ Server ] Error: Event handlers cannot be passed to Client Component props.
  <... placeholder=... onChange={function onChange}>
                                ^^^^^^^^^^^^^^^^^^^
If you need interactivity, consider converting part of this to a Client Component.

Reply:
Ah ja genau, das funktioniert nicht, weil es ein client component ist, müsste man dann auf einer echten Seite anders machen, arbeite ich noch schnell um